app/javascript/src/Policies/styles/policies.scss

```diff
@@ -63,6 +64,25 @@ $subtleColor: #ddd;
     }
   }
 
+  .PolicyConfiguration--preview {
+    overflow: hidden;
+    position: relative;
+
+    &::after {
+      color: $subtleColor; //$label-color;
+      content: 'Preview';
+      font-size: 10rem;
+      left: 4.5rem;
+      opacity: .5;
```

Comment:
We disallow floating decimals in our JS, I think it should do the same here.

Reply:
I think here is an exception, since the opacity property can take a value from 0.0 - 1.0.